client/app/resources/i18n/domains/Messages_fr_FR.xml

```diff
@@ -22,7 +23,7 @@
    <translation id="domains_dashboard_tooltip_whois_multiples" qtlid="143121">Configurer le WHOIS des domaines sélectionnés</translation>
    <translation id="domains_dashboard_tooltip_dnssec_multiples" qtlid="184542">Configurer DNSSEC des domaines sélectionnés</translation>
    <translation id="domains_dashboard_tooltip_export_csv" qtlid="231682">Exporter les domaines en format CSV</translation>
-   <translation id="domains_dashboard_tooltip_export_csv_expl" qtlid="273046">L'export de vos domaines peut durer quelques minutes (en fonction du nombre de domaines que vous possédez). Durant l’opération, vous pouvez utiliser normalement le manager en restant dans la partie web. Le téléchargement va s’éffectuer automatiquement.</translation>
+   <translation id="domains_dashboard_tooltip_export_csv_expl" qtlid="273046">L'export de vos domaines peut durer quelques minutes (en fonction du nombre de domaines que vous possédez). Durant l'opération, vous pouvez utiliser normalement le manager en restant dans la partie web. Le téléchargement va s'éffectuer automatiquement.</translation>
```

Comment:
Why are you doing this type change?

Reply:
`’` isn't a correct apostophe in French so he is right